scripts/config.py

```diff
@@ -172,22 +172,28 @@ def realfull_adapter(_name, active, section):
         return active
     return True
 
-UNSUPPORTED_FEATURE = frozenset([
+PSA_UNSUPPORTED_FEATURE = frozenset([
     'PSA_WANT_ALG_CBC_MAC',
     'PSA_WANT_ALG_XTS',
     'PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_DERIVE',
     'PSA_WANT_KEY_TYPE_DH_KEY_PAIR_DERIVE'
 ])
 
-DEPRECATED_FEATURE = frozenset([
+PSA_DEPRECATED_FEATURE = frozenset([
     'PSA_WANT_KEY_TYPE_ECC_KEY_PAIR',
     'PSA_WANT_KEY_TYPE_RSA_KEY_PAIR'
 ])
 
-UNSTABLE_FEATURE = frozenset([
+PSA_UNSTABLE_FEATURE = frozenset([
     'PSA_WANT_ECC_SECP_K1_224'
 ])
 
+EXCLUDE_FROM_CRYPTO = frozenset(
+    PSA_UNSUPPORTED_FEATURE |
```

Comment:
I agree and also note that the square brackets are missing too. Normally we have `frozenset([...])`

Reply:
The `|` is the union operator for the sets. I create a set from union of sets.

Reply:
But is there a reason for doing it this way, rather than the way it is done elsewhere in the code? Or is it doing something different?

Reply:
In the other places in the code the config names are listed in a list and it is different type than the frozenset. I need to convert one of them to the type of the other and it seemed better solution to just add the element of the set to the list.

Reply:
The weird thing here is that the code is `X = frozenset(A | B | C)` but the argument passed to the `frozenset` constructor is already a frozenset. This could be simplified to `X = A | B | C`.